packages/prerender/package.json

```
@@ -6,13 +6,12 @@
   "files": [
     "dist",
     "dist/index.js",
```

It should be fine to just specify "dist" as a top level, but we should also include "browserUtils"
```suggestion
    "browserUtils"
```